### ISSUE TITLE
Bug On update ?

### ISSUE THREAD

**Issue by ochurlaud** [macaw-movies/macaw-movies#87]
```
In updatePeopleInMovie():
FOREIGN KEY constraint failed Unable to fetch row
```

To explore further

**Comment by ochurlaud** [comment]
I think it's linked to #88

**Comment by sebtouze** [comment]
Not quite shure to see the link with #88. Can you be more explicit on how you get this error ? 
thx

**Comment by ochurlaud** [comment]
The bug appears (I think) when you add an already known person that is not seen as such (because different id).

**Comment by ochurlaud** [comment]
I never seen it again... I close the issue

**Comment by ochurlaud** [comment]
oups... Seen it again.

**Comment by ochurlaud** [comment]
should be corrected with df064eeeef4fcdce68156a2f29ce77112c9dd711 : a people was not processed and was therefore unknown....

Reopen if appears again
